src/Quasar/QuasarF/Interpreter/Internal.purs

```diff
@@ -149,17 +149,33 @@ mkRequest
   . (String → Either Error a)
   → AXF.AffjaxF RequestContent String
   → Free (Coproduct l AXFP) (Either QError a)
-mkRequest f = map (handleResult f) <<< liftF <<< right
+mkRequest f a =
+  map (map snd) $ mkRequest' f a
+
+mkRequest'
+  ∷ ∀ a l
+  . (String → Either Error a)
+  → AXF.AffjaxF RequestContent String
+  → Free (Coproduct l AXFP) (Either QError (Tuple (AX.AffjaxResponse String) a))
+mkRequest' f = map (handleResult f) <<< liftF <<< right
+
+withExpired ∷ ∀ a b. Tuple (AX.AffjaxResponse a) b → { content ∷ b, expired ∷ Boolean }
```

Comment:
`ExpiredContent Boolean`

Reply:
you mean swap `{ content ∷ b, expired ∷ Boolean }` -> `ExpiredContent Boolean` ?

Reply:
yes